src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -115,7 +115,9 @@ function getReactRootElementInContainer(container: any) {
 
 function shouldReuseContent(container) {
   const rootElement = getReactRootElementInContainer(container);
-  return !!(rootElement && rootElement.getAttribute(ID_ATTRIBUTE_NAME));
+  return !!(rootElement &&
+    rootElement.getAttribute &&
```

Comment:
I'd prefer to specifically check the `rootElement.nodeType === 1` here instead of relying on duck typing the method which may exist but do something else and even accept other arguments.